src/Daemon.php

```diff
@@ -67,13 +70,12 @@ public function setListener(callable $listener)
      */
     public function run()
     {
-        $env = $this->context->env;
-        $tube = $env->get('BEANSTALKD_TUBE') ?: 'default';
-        $class = $env->get('BEANSTALKD_CONSUMER');
-        if (!$class) {
+        if (!$this->env->hasValue(self::CONSUMER_KEY)) {
             $this->stdio->outln('<<red>>BEANSTALKD_CONSUMER environmental variable is not set<<reset>>');
             return Status::USAGE;
         }
+
+        $class = $this->env->getValue(self::CONSUMER_KEY);
         if (!class_exists($class)) {
             $this->stdio->outln(sprintf('<<red>>BEANSTALKD_CONSUMER does not reference a locatable class: %s<<reset>>', $class));
```

Comment:
Not sure I follow. The consumer to use is something that will usually be specified from CLI at runtime to allow for the same runner to be used regardless of which consumer was being used.